src/GovUk.Education.ExploreEducationStatistics.Common/Services/ImportStatusService.cs

```diff
@@ -150,16 +150,26 @@ public async Task<bool> UpdateStatus(Guid releaseId, string origDataFileName, IS
             return true;
         }
 
-        public async Task UpdateProgress(Guid releaseId, string origDataFileName, double percentageComplete, int retry = 0)
+        public async Task UpdateProgress(Guid releaseId, string origDataFileName, IStatus status,
```

Comment:
Would it be a good idea to combine both update methods into a single `UpdateStatus` method? I think if we did this, we wouldn't actually need this new status check.

Reply:
Cool, yep, that would clean things up a bit! I have made this change.